splinterd/src/main.rs

```diff
@@ -386,13 +386,14 @@ fn main() {
 
     #[cfg(feature = "rest-api-cors")]
     let app = app.arg(
-        Arg::with_name("whitelist")
```

Comment:
This should probably be an alias for allow_list and hidden in the doc output, for backwards compatibility

Reply:
An alias is already added on line 392